assets/js/race.js

```diff
@@ -312,6 +318,27 @@ function resetIndividual() {
 }
 
 /*
+ * This function post the individual data to the controller.
+ */
+function postIndividual() {
+    var item = {
+        data: data
+    };
+    $.each(item.data, function(order, ind) {
+        ind.ifteam = +ind.ifteam;
+        ind.meal16 = +ind.meal16;
+        ind.meal17 = +ind.meal17;
+    });
+    $.post(controller, item, function(response) {
+        if (response.code != "200") {
+            alert(response.msg);
+            return;
+        }
+    });
+    window.location.assign(directto);
```

Comment:
I would suggest to move this redirect back to click event. `postIndividual()` should only do POST rather than anything else.